util-taglib/src/META-INF/liferay-ui.tld

```diff
@@ -1297,6 +1297,17 @@
 			<required>true</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<name>permissionPropagationCheckboxLabel</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<name>permissionPropagationCheckboxValue</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+			<type>boolean</type>
+		</attribute>
```

Comment:
`Checkbox` in API name is not ideal. In future, we might want to implement this feature with different UI. Maybe something like:
- `permissionPropagationLabel`
- `permissionPropagationEnabled`

Does that reflect meaning of attributes?

Reply:
Yes, those names are good. Thanks for your feedback. I will update the PR.

Reply:
I updated the code. Please take a look again. Thanks.